common/ioshim_windows.go

```diff
@@ -21,10 +22,21 @@ func NewIOShim() *IOShim {
 }
 
 func NewMockIOShim(calls []testutils.TestCmd) *IOShim {
-	return &IOShim{
-		Exec: testutils.GetFakeExecWithScripts(calls),
-		Hns:  hnswrapper.NewHnsv2wrapperFake(),
-	}
+	hns := hnswrapper.NewHnsv2wrapperFake()
+    network := &hcn.HostComputeNetwork{
+        Id:   "1234",
+        Name: "azure",
+    }
+
+    _, err := hns.CreateNetwork(network)
+    if err != nil {
+        return nil
```

Comment:
I know it's a mock, but it still might be nice to put a comment saying why we're tossing the error here. There's a knee-jerk reaction to return all errors, so this seems odd from an eye-glance.

Reply:
At the least will need a log here.

Reply:
added comments, since the network call never returns an error we decided to remove that check